iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/TreeFactory.java

```diff
@@ -255,29 +255,29 @@ public IfExpression ifExpression(SyntaxToken keyword, ParenthesizedExpression co
     return new IfExpressionImpl(keyword, condition, object);
   }
 
-  public Expression memberExpression(Expression value, List<MemberExpression> memberExpressionComponent) {
+  public MemberExpression memberExpression(Expression value, List<MemberExpression> memberExpressionComponent) {
```

Comment:
```suggestion
  public MemberExpression memberExpression(Expression value, List<MemberExpression> memberExpressionComponents) {
```
I think that having variables for collections named as plural somewhat improves readability, WDYT?

Reply:
Definitely